selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -50,21 +50,18 @@ initVariables() {
     readonly FAILSAFE_REPORT="target/site/failsafe-report.html"
 
     readonly SINGLE_TEST_MSG="single test/package"
-
-    readonly MAX_RERUN=2
+    export CHE_MULTIUSER=${CHE_MULTIUSER:-false}
 
     # CALLER variable contains parent caller script name
     # CUR_DIR variable contains the current directory where CALLER is executed
     [[ -z ${CALLER+x} ]] && { CALLER=$(basename $0); }
     [[ -z ${CUR_DIR+x} ]] && { CUR_DIR=$(cd "$(dirname "$0")"; pwd); }
 
     [[ -z ${API_SUFFIX+x} ]] && { API_SUFFIX="/api/"; }
-    [[ -z ${BASE_ACTUAL_RESULTS_URL+x} ]] && { BASE_ACTUAL_RESULTS_URL="https://ci.codenvycorp.com/view/qa/job/che-integration-tests-che6/"; }
 
     MODE="grid"
     GRID_OPTIONS="-Dgrid.mode=true"
-    RERUN=false
-
+    RERUN_ATTEMPTS=0
```

Comment:
Why did you change the default value to `0` ?

Reply:
It means we DO NOT rerun tests if `--rerun` parameter hasn't bee set.

Reply:
ok. So, let's automatically set `RERUN_ATTEMPTS` to `1` if `rerun` is set.
If a developer needs more attempts then he has to set it explicitly.

Reply:
Good advice, thank you!